lib/rubycritic/core/analysed_module.rb

```diff
@@ -23,6 +23,10 @@ def cost
       @cost ||= smells.map(&:cost).inject(0, :+) + (complexity / 25)
     end
 
+    def limited_cost
+      cost < 32 ? cost : 32
```

Comment:
Magic number - why 32? Literal duplication as well. How about making this a constant?